flujo/cli/generators/openapi_agent_generator.py

```
@@ -0,0 +1,219 @@
+        base_url=base_url,
+        url_path=url_path,
+        query=query,
+        headers=headers,
+        json_body=body,
+    )
+"""
+        )
+
+    func_list = ", ".join(_safe_name(op.get("operationId") or f"{m}_{p}") for p, m, op in ops)
+    resp_map_lines = "\n".join(
+        f'    "{name}": {model if model in {"dict", "list"} else model},'
```

Ruff (0.14.7)

Useless `if`-`else` condition

(RUF034)